app/views/spree/components/checkout/_payment_controls.html.erb

```diff
@@ -0,0 +1,10 @@
+<ul class="payment-controls" id="payment-methods">
```

Comment:
This `id` looks a bit useless to me. Maybe we can remove it?

Reply:
The id payment-methods is used in js to show/hide payment methods [here](https://github.com/nebulab/solidus_starter_frontend/blob/f139a08923e16c9b8ded26594c0c793f9e5cd3ec/app/assets/javascripts/spree/frontend/checkout/payment.js#L6)

Reply:
Can't we use a class in the JS? I'm also ok making this in another PR but please open an issue or we'll forget about it.